substratevm/src/com.oracle.svm.test/src/hello/Hello.java

```diff
@@ -229,4 +229,6 @@ private static void inlineReceiveConstants(byte b, int i, long l, String s, floa
         System.out.println(String.format("q = %g\n", q));
         System.out.println(String.format("t = %s\n", t));
     }
+
+    final static java.util.function.Supplier<String> lambda = () -> "lambdaText";
```

Comment:
I do not see any use of that lambda elsewhere. Thus, it is not guaranteed that the reachability analysis will keep it.

Reply:
Yes, this actually needs to be called by the test program.